Kommunicate/Classes/KMConversationListViewController.swift

```diff
@@ -400,15 +401,21 @@ public class KMConversationListViewController : ALKBaseViewController, Localizab
             case .success(let conversationId):
                 DispatchQueue.main.async {
                     self.view.isUserInteractionEnabled = true
-                    alertView.dismiss(animated: false, completion: nil)
-                    Kommunicate.showConversationWith(groupId: conversationId, from: self, completionHandler: { (success) in
-                        print("Conversation was shown")
-                    })
+                    alertView.dismiss(animated: true, completion: nil)
                 }
+                let applozicClient = self.applozicClientType.init(applicationKey: KMUserDefaultHandler.getApplicationKey())
```

Comment:
Can you use `ALChannelService` to maintain consistency? And we can dismiss the activity indicator after getting the info.